components/bl-gallery-component/src/components/gallery-image.js

```diff
@@ -0,0 +1,11 @@
+export function GalleryImage({ url, imageHeight }) {
+  return (
+    <>
+      <img
+        src={ url }
+        style={{ height: imageHeight ? imageHeight : '' }}
```

Comment:
since the component is called `...Image` it doesn't need to add `image` word to properties names

just `height` will look better